### DISCUSSION TITLE
Firetv

### DISCUSSION THREAD

**Discussion by parvbarot** [Nerwyn/universal-remote-card#26] [General]
Does this awesome card work with Fire TV?

**Comment by Nerwyn** [comment]
From the [Android TV Remote integration page](https://www.home-assistant.io/integrations/androidtv_remote/#limitations-and-known-issues), it looks like this card won't work with it out of the box.

> The integration doesn’t work with Fire TV devices because they are missing the [Android TV Remote Service](https://play.google.com/store/apps/details?id=com.google.android.tv.remote.service). Attempts to sideload it haven’t been successful.

Home Assistant can control [Fire TVs](https://www.home-assistant.io/integrations/fire_tv/) via the [Android Debug Bridge](https://www.home-assistant.io/integrations/androidtv), which this card also uses for the keyboard. You could set that up and then create custom keys and sources for [all of the Fire TV commands](https://www.home-assistant.io/integrations/androidtv#services).
